libraries/AP_JSONParser/jsmn.h

```diff
@@ -0,0 +1,475 @@
+// this is the "JSMN" ("Jasmine") JSON parser.
+// it was directly copied from the github page here: https://raw.githubusercontent.com/zserge/jsmn/master/jsmn.h
+// thanks @Serge
```

Comment:
Can you add a commit hash or something so we know which version you pulled?

Reply:
Done, thanks!